.github/pull_request_template.md

```diff
@@ -0,0 +1,9 @@
+## Description
+- Description of change
+
+## Detailed Changes
+
+## Additional Info
+- Type of PR (bugfix, feature)
+- Fixes (issue link)
+- Screenshots (if any)
```

Comment:
Lets tackle this in a separate PR.
Lets keep this PR focused on the issues template